public/stylesheets/sdil.css

```diff
@@ -30,6 +30,9 @@
     color: #4c2c92;
 }
 
+.change-link-padding {
```

Comment:
This is you issue with the change link positioning. Also the underline on the link looks thicker than other buttons, can you change this please?